data/hai/midas.txt

```diff
@@ -0,0 +1,940 @@
+		payment 1000000
+	# as of yet, we cannot make the main mission fail as your escorts are boarded, meaning these will not be disables
+	npc
+		# the Alphas and their minions left Midas alone so that this would happen. They probably even gave them a tip about it, but not in person.
+		government "Midas"
+		system "Heia Due"
+		personality plunders staying harvests
+		fleet "Large Northern Pirates" 6
+		on provoke
+			dialog `You hear a voice once alone in your quarters, "You have chosen your side. Nemo Me Impune Lacessit, <first>." You feel a strong pain in your neck.`
+			"reputation: Midas" <?= -1000
+			"midas: wronged" ++
+	npc
+		government "Pirate"
+		system "Heia Due"
+		personality entering plunders staying
+		fleet "Small Northern Pirates" 10
+	to fail
+		or
+			has "<uhai> 6 A: Loyal Escort: done"
+			has "<uhai> 6 A: Loyal Escort: aborted"
+	on fail
+		dialog `You hear a voice once alone in your quarters, "A shame, captain. Truly a shame."`
+	to complete
+		has "<uhai> 6 A: Loyal Escort: failed"
+	on complete
+		"unfettered: upset Sookootow" ++
+		"reputation: Midas" <?= -1000
+		clear "unfettered: stopped on Greenbloom"
+		log `The plan was to escort a Hai fleet carying supplies and passengers into Unfettered space but an opportunity to feed them to the wolves instead was presented, and taken.`
+		payment 4000000
+		dialog
+			`As before, a voice addresses you once you are alone in your quarters. "We have transferred 4 million credits to your account, and made sure the Chief will not bother you.`
+			`	"Do be aware, however, that a food situation has arrisen, not because of our small operation, but because of the bomb's presence in the first place.`
+			`	"The Unfettered will probably be busy for some time... As for the governor, if it can ease your conscience, we only captured her."`
+
+
+
+# secret mission
+# happens if you accept a duel in "<uhai> 7: Stolen Cargo" (offered if your fleet is smaller than 2 heavy warships)
+mission "<uhai> 7: Duel"
+	landing
+	source "New Tortuga"
+	name "Duel with the twins."
+	description "Beat the two twins today, or lose 10 million credits!"
+	to offer
+		has "midas: accepted duel"
+		has "<uhai> 7: Cavalry Arrives: done"
+		"unfettered: cleaned up cerberus" > 1
+	on offer
+		dialog `The two mysterious ships hail you, "We are ready for a fight in orbit whenever you are. If you want, you can still call this off... for a quarter of the agreed price."`
+	on decline
+		fine 2500000
+	npc disable save
+		government "Hai (Unfettered Challenger)"
+		to despawn
+			has "despawn the duel ships"
+		personality waiting heroic unconstrained staying disables
+		ship "Sea Dragon" "Thorn"
+		ship "Sea Dragon" "Talon"
+		on kill
+			dialog "You killed one of the ships. You will need to restart the mission in order to progress further with the Shadow of Mintaka. They may want to have their vengeance on you... The Unfettered may not offer you any missions anymore, for disrespecting the sacred rules of the duel."
```

Comment:
I think instead of telling the player to restart the mission, you could say something similar to FW fail messages. Just my opinion, though.

Reply:
Thats the thing, this is an optional mission chain that will come later
Killing the ships wont lock you out of uhai

Reply:
If you say `You will need to restart the mission in order to progress further with the Shadow of Mintaka.` then, chances are, people are going to reload a save. Since it leads to another storyline, I would change the line.

Reply:
killing them actually triggers mommy to come and kill you and thats the only storyline for them rn x)

Reply:
```suggestion
			dialog "Killing at least one of the twins will have consequences with the Shadow of Mintaka. As they said, "nemo me impune lacessit!" The Unfettered may refrain from offering you missions, for disrespecting the sacred rules of the duel."
```
the big deal is here you killed them in the middle of a duel which is bad, very bad! ~~prob should have consequences~~

Reply:
Maybe, I just wish there was a clearer way to say there's another branch this could lead to without saying it plainly.

Reply:
If you have suggestions, shoot
Otherwise the one I wrote is less worse, right?

Reply:
I can't think of anything now, yeah it's a little better. If I think of something later I'll just start a new conversation.